### PR TITLE
Existing Request page gray boxes

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#5781

*Description of changes:*

This is part 1 of this ticket. A future PR will have the tooltip changes.

- added Check Status gray box
- added NR Approved gray box
- added NR Not Approved gray box
- added Contact Info component
- added code to display gray boxes
- added some global text styles
- updated definition of "conditional" to include COND_RESERVE
- updated COND RESERVED to display as Approved
- added enum for Entity Code
- updated some NR State enums
- updated bcregistries -> BCRegistries
- renamed NamesBlock -> NamesGrayBox
- fixed gutter classes on misc gray boxes
- moved some getters to common mixin
- updated footer Phone per Scott
- renamed Entity Code enum -> Entity Type
- misc cleanup

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/existing-request/existing-request-display.vue [hunk 1]
```diff
@@ -247,6 +279,31 @@ export default class ExistingRequestDisplay extends Mixins(NrAffiliationMixin, C
...
+  private get showNrNotApproved (): boolean {
+    return (this.nr.state === NrState.REJECTED)
+  }
+
+  /** True if this is a Benefit Company NR. */
+  private get isBenefitCompany (): boolean {
+    return (this.nr.entity_type_cd === EntityCode.BC)
+  }
+
+  /** Whether this NR is in a conditional state. */
```
Comment (severinbeauvais) [inline client/src/components/existing-request/existing-request-display.vue:302]: change "whether" to "true if"

File: client/src/components/existing-request/check-status-gray-box.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<template>
+  <!-- TODO: try a v-row with prop "no-gutters" -->
```
Comment (severinbeauvais) [inline client/src/components/existing-request/check-status-gray-box.vue:2]: todo

File: client/src/components/existing-request/nr-not-approved-gray-box.vue [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import ContactInfo from '@/components/common/contact-info.vue'
+
+@Component({
+  components: { ContactInfo }
+})
+export default class NrNotApprovedGrayBox extends Vue {
+  @Prop({ default: 'TBD' })
+  readonly nrNum: string
+
+  @Prop({ default: 'TBD' })
```
Comment (severinbeauvais) [inline client/src/components/existing-request/nr-not-approved-gray-box.vue:23]: delete these

File: client/src/enums/entity-code.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+/** Enum for Entity Code. */
+export enum EntityCode {
```
Comment (severinbeauvais) [inline client/src/enums/entity-code.ts:2]: rename to Entity Type

File: client/src/enums/nr-action.ts [hunk 5]
```diff
@@ -5,7 +5,8 @@ export enum NrAction {
   REAPPLY = 'REAPPLY',
   RECEIPTS = 'RECEIPT',
   REFUND = 'REQUEST_REFUND',
-  RESULTS = 'RESULT', // NOT YET IMPLEMENTED
+  RESEND = 'RESEND', // FUTURE: will be removed
```
Comment (severinbeauvais) [inline client/src/enums/nr-action.ts:8]: will be removed now

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by lekshmimallika-aot — APPROVED** [review]
Nice!

**Review by cameron-freshworks — APPROVED** [review]
Looks great!

**Pull request merged**